python/test/unit/runtime/test_cache.py

```diff
@@ -553,28 +554,37 @@ def compiled_hook(*args, **kwargs):
 
 @pytest.mark.skipif(reason="within_2g is a HIP specific optimization", condition=not is_hip())
 def test_within_2gb(device, fresh_triton_cache) -> None:
+    from triton.backends.amd.compiler import use_buffer_ops
+    default_buffer_ops = os.environ.get("AMDGCN_USE_BUFFER_OPS", "0")
+    try:
+        # Set AMDGCN_USE_BUFFER_OPS and invalidate the cache
+        use_buffer_ops.cache_clear()
```

Comment:
This isn't actually clearing the cache because the test and module have a different copy of the lru_cache object. I need to investigate this further.